app/controllers/jobs_controller.rb

```diff
@@ -15,26 +18,24 @@ def new
     @job = Job.new
   end
 
-  def create
-    uploaded_file = params[:file]
-    form = UploadForm.new(file: uploaded_file)
-    form.validate!
+  def sign
+    filename     = params[:filename].presence || "upload/#{SecureRandom.uuid}"
+    content_type = params[:content_type].presence || 'application/pdf'
 
     job = current_user.jobs.build
     job.status = 'processing'
     job.uuid = SecureRandom.uuid
     job.save!
+    object_key = create_object_key(filename)
+    s3_handler = S3Handler.new(object_key)
+    s3_handler.bucket
+    render json: s3_handler.presigned_url_for_input(filename, content_type, job.id)
+  end
 
-    persistent_path = form.persist_to_tmp!
-
-    GUIRemediationJob.perform_later(job.uuid, persistent_path, uploaded_file.original_filename)
-
-    redirect_to job_path(job), notice: I18n.t('upload.success')
-  rescue ActiveModel::ValidationError
-    flash[:alert] = I18n.t('upload.error')
-    redirect_to action: 'new'
-  rescue StandardError => e
-    flash[:alert] = "Exception: #{e.message}"
-    redirect_to action: 'new'
+  def complete
+    job = Job.find(params[:job_id])
+    output_url = params[:output_url]
+    output_object_key = params[:output_object_key]
+    update_job(job, output_url, output_object_key)
```

Comment:
Sorry, I spoke to soon.  It _appears_ to work locally, but looking here, and at the filename of the "completed" file, it looks like we are not running the `GUIRemediationJob` and not retrieving the remediated file.  It also looks like the code for `GUIRemediationJob` has been deleted.  We still need to queue that job and run it to poll the output directory in S3 to grab the remediated file.  I think this controller action just needs tweaked to kickoff that job.  Or, I may be missing something about how this works.